src/types_.py

```diff
@@ -9,6 +9,25 @@
 from pathlib import Path
 from urllib.parse import urlparse
 
+
+class Metadata(typing.NamedTuple):
+    """Information within metadata file. Refer to: https://juju.is/docs/sdk/metadata-yaml.
+
+    Attrs:
+        name: Name of the charm.
+
```

Comment:
Is this intentionally a blank line?

Reply:
I wanted to seperate the required/optional fields but shoudl be resolved with the comments below! Thank you!